vktotg.py

```diff
@@ -1,6 +1,8 @@
+#!/usr/bin/python3
```

Comment:
please, add to README `pip3`. because `pip install telethon`  fails due to syntax error